benchmark.js

```diff
@@ -0,0 +1,12 @@
+const siege = require('siege');
+
+let stress = siege().on(3003).concurrent(15);
+
+const randomRoomId = (min, max) => Math.floor(Math.random() * Math.floor(max - min + 1)) + min;
+
+for (let i = 1; i < 100000; i += 1) {
+  const roomId = randomRoomId(1, 10000000);
```

Comment:
Small thing, but we're supposed to target latter 10% of dataset

Reply:
Good catch! I just pushed up the update now. Thanks!